pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Ah I see, that makes sense. It is still possible to extend your privileges by updating the role itself, but it seems the intent here is to make it possible to delegate permission management to a user who can create/update role bindings, but not roles themselves, is that correct?